src/main/java/org/thymeleaf/templateresolver/JakartaServletContextTemplateResolver.java

```diff
@@ -0,0 +1,64 @@
+/*
```

Comment:
If I'm right, I believe the `org.thymeleaf.web.servlet.*` approach would make the duplication of template/resource resolvers also unneeded…

Reply:
This is true. However, I think the disadvantages I listed above outweigh the benefits.

We could extract a base class that removes any duplication between the resources.